ratelimits/limit_test.go

```diff
@@ -9,6 +9,32 @@ import (
 	"github.com/letsencrypt/boulder/test"
 )
 
+// loadAndParseDefaultLimits is a helper that calls both loadDefaults and
+// parseDefaultLimits to handle a YAML file.
+//
+// TODO: Update the tests to test these functions individually.
+func loadAndParseDefaultLimits(path string) (limits, error) {
+	fromFile, err := loadDefaults(path)
+	if err != nil {
+		return nil, err
+	}
+
+	return parseDefaultLimits(fromFile)
+}
+
+// loadAndParseOverrideLimits is a helper that calls both loadOverrides and
+// parseOverrideLimits to handle a YAML file.
+//
+// TODO: Update the tests to test these functions individually.
```

Comment:
We should include an issue with every TODO, in the form of:

```suggestion
// TODO(#12345): Update the tests to test these functions individually.
```